xeofs/models/_base_cross_model.py

```diff
@@ -118,6 +139,13 @@ def __init__(
             else None
```

Comment:
When using PCA-preprocessed MCA, checking for `NaN`s is not necessary anymore since the input data has already been `Preprocessor`-processed. So we can just setting it to `False`.


```suggestion
        self.pca1 = (
            EOF(
                n_modes=n_pca_modes,
                compute=self._params["compute"],
                check_nans=False,
            )
            if n_pca_modes
            else None
        )
        self.pca2 = (
            EOF(
                n_modes=n_pca_modes,
                compute=self._params["compute"],
                check_nans=False,
                )
            if n_pca_modes
            else None
        )
```